phys2bids/viz.py

```diff
@@ -41,3 +41,25 @@ def ntr2time(x):
     subplot.plot(time, trigger, '-', time, time, '-')
     plt.savefig(outfile + '_trigger_time.png', dpi=dpi)
     plt.close()
+
+
+def plot_all(phys_in, infile, outfile='', dpi=SET_DPI, size=FIGSIZE):
+    ch_num = len(phys_in.ch_name)  # get number of channels:
+    fig, ax = plt.subplots(ch_num - 1, 1, figsize=size, sharex=True)
+    time = phys_in.timeseries[0]  # assume time is first channel
+    fig.suptitle(os.path.basename(infile))
+    for row, timeser in enumerate(phys_in.timeseries[1:]):
+        if timeser.shape != time.shape:
+            time_old = np.linspace(0, time[-1], num=timeser.shape[0])
+            timeser = np.interp(time, time_old, timeser)
```

Comment:
Well guys here you have it, interpolation with np.interp. I already delete any presence of scipy so this should be now ready for merging

Reply:
Great! Thanks @vinferrer !